api/api_sources/sources/database/migrations/1562358560325-CreateObservationCodes.ts

```diff
@@ -60,16 +60,16 @@ export class CreateObservationCode1562358560325 implements MigrationInterface {
         await queryRunner.query(this.aspectCodeSchema.migrationSQL);
         await queryRunner.query(this.proposedCodeSchema.migrationSQL);
         // Pre-load codes
-        await queryRunner.query(getSQLFileData(this.densitySchema.dataSQLPath()));
-        await queryRunner.query(getSQLFileData(this.distributionSchema.dataSQLPath()));
-        await queryRunner.query(getSQLFileData(this.agencyCodeSchema.dataSQLPath()));
-        await queryRunner.query(getSQLFileData(this.observationTypeCodeSchema.dataSQLPath()));
-        await queryRunner.query(getSQLFileData(this.soilTextureCodeSchema.dataSQLPath()));
-        await queryRunner.query(getSQLFileData(this.observationGeometryCodeSchema.dataSQLPath()));
-        await queryRunner.query(getSQLFileData(this.specificUseCodeSchema.dataSQLPath()));
-        await queryRunner.query(getSQLFileData(this.slopeCodeSchema.dataSQLPath()));
-        await queryRunner.query(getSQLFileData(this.aspectCodeSchema.dataSQLPath()));
-        await queryRunner.query(getSQLFileData(this.proposedCodeSchema.dataSQLPath()));
+        await queryRunner.query(getSQLFileData(this.densitySchema.dataSQLPath(), this.densitySchema.className));
+        await queryRunner.query(getSQLFileData(this.distributionSchema.dataSQLPath(), this.distributionSchema.className));
+        await queryRunner.query(getSQLFileData(this.agencyCodeSchema.dataSQLPath(), this.agencyCodeSchema.className));
+        await queryRunner.query(getSQLFileData(this.observationTypeCodeSchema.dataSQLPath(), this.observationTypeCodeSchema.className));
+        await queryRunner.query(getSQLFileData(this.soilTextureCodeSchema.dataSQLPath(), this.soilTextureCodeSchema.className));
+        await queryRunner.query(getSQLFileData(this.observationGeometryCodeSchema.dataSQLPath(), this.observationTypeCodeSchema.className));
```

Comment:
shouldn't this be "this.observationGeometryCodeSchema.className" ?

Reply:
Fixed

Reply:
Fixed